presto-base-jdbc/src/test/java/com/facebook/presto/plugin/jdbc/TestJdbcClient.java

```diff
@@ -110,4 +117,43 @@ public void testMetadataWithFloatAndDoubleCol()
                 new JdbcColumnHandle(CONNECTOR_ID, "COL3", JDBC_DOUBLE, DOUBLE),
                 new JdbcColumnHandle(CONNECTOR_ID, "COL4", JDBC_REAL, REAL)));
     }
+
+    @Test
+    public void testCreateTable()
+    {
+        String tableName = randomUUID().toString().toUpperCase(Locale.getDefault());
+        SchemaTableName schemaTableName = new SchemaTableName("exa_ple", tableName);
```

Comment:
I'd rather use the `example` schema for these, since we aren't testing listing behavior (where underscore matters as it is a pattern character). Using `exa_ple` here looks strange and thus detracts from readability.